upgrade-agent/upgradeproto/upgrade.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package upgrade;
+
+option go_package = "github.com/edgelesssys/constellation/v2/upgrade-agent/upgradeproto";
+
+service Update {
+    rpc ExecuteUpdate(ExecuteUpdateRequest) returns (ExecuteUpdateResponse);
+}
+
+message ExecuteUpdateRequest {
+    string kubeadm_URL = 1;
+    string kubeadm_Hash = 2;
+    string wanted_Kubernetes_Version = 3;
```

Comment:
```suggestion
    string kubeadm_url = 1;
    string kubeadm_hash = 2;
    string wanted_kubernetes_version = 3;
```

Sorry, this is my fault. I haven't merged the correction yet, but it's part of https://github.com/edgelesssys/constellation/pull/716.